versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CacheBackend.java

```diff
@@ -44,6 +52,12 @@ public interface CacheBackend {
   /** Adds the given object only to the local cache, does not send a cache-invalidation message. */
   void putLocal(@Nonnull String repositoryId, @Nonnull Obj obj);
 
+  /**
+   * Record the "not found" sentinel for the given {@link ObjId id} and {@link ObjType type}.
+   * Behaves like {@link #remove(String, ObjId)}, if {@code type} is {@code null}.
+   */
+  void putNegative(@Nonnull String repositoryId, @Nonnull ObjId id, @Nonnull ObjType type);
```

Comment:
nit: why not keep it close (line-wise) to the other `putNegative` method?

Reply:
it's reference related - renamed it